src/main/java/bio/terra/service/cohortbuilder/tanagra/configuration/TanagraExportConfiguration.java

```diff
@@ -0,0 +1,46 @@
+package bio.terra.service.cohortbuilder.tanagra.configuration;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@EnableConfigurationProperties
+@ConfigurationProperties(prefix = "tanagra.export")
+public class TanagraExportConfiguration {
```

Comment:
I don't think export is needed for our cohort builder so this should be removable at some point.